### ISSUE TITLE
No registry provided for decentralized stranger node verification 

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#2469]
**Describe the Bug**

During node discovery Ursula reports `No registry provided for decentralized stranger node verification `.  This indicated that some ursulas are not validated as stakers in the escrow contract during node discovery.

**To Reproduce**

`nucypher ursula run` or `nucypher alice grant` observing the logs.

**Logs**

```
2020-12-10T10:54:28-0800 [LocalFileBasedNodeStorage#debug] Saved TLS certificate for DarkSeaGreen X-ray Chartreuse Echo 0xd7d97BbAdd2c1B5b4f08C6F77A478FaF3252D9aC: /home/kieran/.local/share/nucypher/known_nodes/certificates/0xd7d97BbAdd2c1B5b4f08C6F77A478FaF3252D9aC.pem
2020-12-10T10:54:28-0800 [nucypher.characters.lawful.Ursula#debug] No registry provided for decentralized stranger node verification - on-chain Staking verification will not be performed.
2020-12-10T10:54:31-0800 [Alice#debug] Cycled teachers; New teacher is (NodeSprout)⇀DarkMagenta Zulu Gray Mike↽ (0x00002F86f0a0E84210e2E5Bf16E2dA444cd4275C)
2020-12-10T10:54:32-0800 [Alice#info] Learning round 1.  Teacher: (Ursula)⇀DarkSeaGreen X-ray Chartreuse Echo↽ (0xd7d97BbAdd2c1B5b4f08C6F77A478FaF3252D9aC) knew about 1969 nodes, 1970 were new.
2020-12-10T10:54:32-0800 [nucypher.characters.lawful.Ursula#debug] Initialized Stranger Ursula | (Ursula)⇀SlateBlue Juliett SeaShell Hotel↽ (0x5cF1703A1c99A4b42Eb056535840e93118177232)
2020-12-10T10:54:32-0800 [LocalFileBasedNodeStorage#debug] Saved TLS certificate for SlateBlue Juliett SeaShell Hotel 0x5cF1703A1c99A4b42Eb056535840e93118177232: /home/kieran/.local/share/nucypher/known_nodes/certificates/0x5cF1703A1c99A4b42Eb056535840e93118177232.pem
2020-12-10T10:54:32-0800 [nucypher.characters.lawful.Ursula#debug] No registry provided for decentralized stranger node verification - on-chain Staking verification will not be performed.
2020-12-10T10:54:33-0800 [nucypher.policy.policies.BlockchainPolicy#debug] Arrangement accepted by (Ursula)⇀SlateBlue Juliett SeaShell Hotel↽ (0x5cF1703A1c99A4b42Eb056535840e93118177232)
2020-12-10T10:54:33-0800 [nucypher.characters.lawful.Ursula#debug] Initialized Stranger Ursula | (Ursula)⇀FireBrick Papa SkyBlue Five↽ (0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752)
2020-12-10T10:54:33-0800 [LocalFileBasedNodeStorage#debug] Saved TLS certificate for FireBrick Papa SkyBlue Five 0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752: /home/kieran/.local/share/nucypher/known_nodes/certificates/0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752.pem
2020-12-10T10:54:33-0800 [nucypher.characters.lawful.Ursula#debug] No registry provided for decentralized stranger node verification - on-chain Staking verification will not be performed.
2020-12-10T10:54:34-0800 [nucypher.policy.policies.BlockchainPolicy#debug] Arrangement accepted by (Ursula)⇀FireBrick Papa SkyBlue Five↽ (0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752)
2020-12-10T10:54:34-0800 [nucypher.characters.lawful.Ursula#debug] Initialized Stranger Ursula | (Ursula)⇀DarkKhaki Quebec Thistle One↽ (0xbdA6Ba3Ad624582495C55451A1840e563fc16389)
2020-12-10T10:54:34-0800 [LocalFileBasedNodeStorage#debug] Saved TLS certificate for DarkKhaki Quebec Thistle One 0xbdA6Ba3Ad624582495C55451A1840e563fc16389: /home/kieran/.local/share/nucypher/known_nodes/certificates/0xbdA6Ba3Ad624582495C55451A1840e563fc16389.pem
2020-12-10T10:54:34-0800 [nucypher.characters.lawful.Ursula#debug] No registry provided for decentralized stranger node verification - on-chain Staking verification will not be performed.
```

**System**
 - OS Version: Ubuntu 20.04
 - Nucypher Version: 4.3.0
 - Ethereum Node Version: Infura

**Comment by KPrasch** [comment]
Related to #2300

**Comment by KPrasch** [comment]
Fixed before release 4.6.x and further advanced by #2560.  I have been observing a mainnet worker's logs and there is no reproduction of this issue.
